src/InnerEventStore.php

```diff
@@ -48,6 +48,24 @@ public function loadAggregateEvents(string $streamName, string $aggregateType, s
         return $this->eventStore->loadAggregateEvents($streamName, $aggregateType, $aggregateId, $minVersion);
     }
 
+    /**
+     * @param string $streamName
+     * @param string $aggregateType
+     * @param string $aggregateId
+     * @param int $maxVersion
+     * @param int $minVersion
+     * @return \Iterator GenericEvent[]
+     */
+    public function loadAggregateEventsUntil(
```

Comment:
latest version of event store interface does not have the method.

Reply:
Is solved